101-recovery-services-vm-workload-backup/azuredeploy.json

```diff
@@ -0,0 +1,193 @@
+      }
+    },
+    "protectedItemType": {
+      "type": "string",
+      "defaultValue": "AzureVmWorkloadSQLDatabaseProtectedItem",
+      "allowedValues": [
+        "AzureVmWorkloadSQLDatabaseProtectedItem"
+      ],
+      "metadata": {
+        "description": "Protected Item (Database) type"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "variables": {
+    "skuName": "RS0",
+    "skuTier": "Standard",
+    "backupFabric": "Azure",
+    "containerType": "VMAppContainer",
+    "backupManagementType": "AzureWorkload"
+  },
+  "resources": [
+    {
+      "condition": "[parameters('isNewVault')]",
+      "type": "Microsoft.RecoveryServices/vaults",
+      "apiVersion": "2018-01-10",
+      "name": "[parameters('vaultName')]",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "[variables('skuName')]",
+        "tier": "[variables('skuTier')]"
+      },
+      "properties": {}
+    },
+    {
+      "type": "Microsoft.RecoveryServices/vaults/backupFabrics/protectionContainers",
+      "apiVersion": "2018-01-10",
+      "name": "[concat(parameters('vaultName'), '/', variables('backupFabric'), '/', variables('containerType'), ';compute;', parameters('vmResourceGroup'),';',parameters('vmName'))]",
+      "dependsOn": [
+        "[resourceId('Microsoft.RecoveryServices/vaults/', parameters('vaultName'))]"
+      ],
+      "properties":{
+        "containerType": "[variables('containerType')]",
+        "backupManagementType": "[variables('backupManagementType')]",
+        "workloadType": "[parameters('workloadType')]",
+        "friendlyName": "[parameters('vmName')]",
+        "sourceResourceId": "[resourceId(subscription().subscriptionId, parameters('vmResourceGroup'),'Microsoft.Compute/virtualMachines', parameters('vmName'))]"
+      }
+    },
+    {
+      "condition": "[parameters('isNewPolicy')]",
+      "type": "Microsoft.RecoveryServices/vaults/backupPolicies",
+      "apiVersion": "2018-01-10",
+      "name": "[concat(parameters('vaultName'), '/', parameters('policyName'))]",
+      "dependsOn": [
+        "[resourceId('Microsoft.RecoveryServices/vaults/', parameters('vaultName'))]"
+      ],
+      "properties": {
+        "backupManagementType": "[variables('backupManagementType')]",
+        "workloadType": "[parameters('workloadType')]",
+        "settings": {
+          "timeZone": "UTC",
+          "issqlcompression": false,
+          "isCompression": false
+        },
+        "subProtectionPolicy": [
+          {
+            "policyType": "Full",
+            "schedulePolicy": {
+                "schedulePolicyType": "SimpleSchedulePolicy",
+                "scheduleRunFrequency": "Daily",
+                "scheduleRunTimes": [
+                    "2019-03-24T16:00:00Z"
```

Comment:
what is this used for?

Reply:
This represents the time when scheduled backups are triggered

Reply:
Should it be hardcoded or start when the template is deployed?

Reply:
This should be hardcoded, not dependent on time of deployment.